api-gateway/src/main/resources/application-dev.yml

```diff
@@ -58,6 +60,6 @@ spring:
             client-secret: 2c05929edd949d670c3454258bbdbfefb7ba3eb7
 logging:
   level:
-    org.springframework.web: DEBUG
-    org.springframework.security: DEBUG
+    org.springframework.web: TRACE
```

Comment:
This will be a lot of logging, maybe make it `DEBUG` again?

Reply:
Yes, sure, my bad

Reply:
Done